pages/docs/reference/js-project-setup.md

```diff
@@ -383,7 +386,13 @@ config.module.rules.push({
 All webpack configuration
 capabilities are well described in its [documentation](https://webpack.js.org/concepts/configuration/).
 
-For building executable JavaScript artifacts though webpack, the Kotlin/JS plugin contains the `browserDevelopmentWebpack` and `browserProductionWebpack` Gradle tasks. Execute them to obtain artifacts for development or production. The final generated artifacts will be available in `build/distributions` unless [specified otherwise](#distribution-target-directory).
+For building executable JavaScript artifacts through webpack, the Kotlin/JS plugin contains the `browserDevelopmentWebpack` and `browserProductionWebpack` Gradle tasks.
```

Comment:
In mpp projects tasks have an additional prefix (by default "js").
Not sure where it's better to mention, here or somewhere else.

Reply:
Also, in mpp projects `run` works differently.

Reply:
This page in general describes the `kotlin.js` plugin. Adding details about using JS targets in MPP requires more updates, so I'd leave it as is for now and maybe form the scope of MPP-JS related updates later.